.github/workflows/main.yml

```diff
@@ -115,6 +116,8 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
+        with:
+          ref: r2020.8
```

Comment:
This isn't strictly necessary for this PR, but I added it to see if it will push up an automated commit to the correct branch.